src/notification.cpp

```diff
@@ -606,10 +843,21 @@ void Notification::setItemCount(int itemCount)
 /*!
     \qmlmethod void Notification::publish()
```

Comment:
Ah, ignore that, when I read the documentation I thought it meant an id had to be passed in, but that's not the case.